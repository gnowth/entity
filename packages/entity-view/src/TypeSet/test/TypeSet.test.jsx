import renderer from 'react-test-renderer';
import React from 'react';
import { Fields } from '@entity/core';
import { IntlProvider } from 'react-intl';

import ViewTypeSet from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <IntlProvider locale="en" messages={{}}>
      <ViewTypeSet
        field={new Fields.CharField()}
        theme={{}}
        value="test"
      />
    </IntlProvider>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
