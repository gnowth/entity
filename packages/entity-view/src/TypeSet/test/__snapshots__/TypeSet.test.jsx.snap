// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<pre
  className="TypeSet-sc-2d00f5-0 gXqwhy"
  value="test"
>
  test
</pre>
`;
